src/implementation.js

```diff
@@ -0,0 +1,76 @@
+// 'Game with ID: 1B9J4M3JIbsNXfnHvmVq added.'
+// CPxigm7hasQ5RkhAz8Il added.'
+
+const user = document.querySelector('.user');
+const score = document.querySelector('.Score');
+
+const renderData = (elements) => {
+  const scores = document.querySelector('.scores');
+  const score = document.createElement('p');
+  score.classList.add('m-0', 'p-2', 'score');
+  score.innerHTML = `${elements.user}: ${elements.score}`;
+  scores.appendChild(score);
+  // scores.innerHTML = `<p class="score m-0 p-2">${elements.user}:${elements.score}</p>`
```

Comment:
- [ ] Kindly remove commented code from the codebase to make your code clean and readable.